stargazer/stargazer.py

```diff
@@ -200,6 +209,28 @@ def reset_covariate_order(self):
         if self.original_cov_names is not None:
             self.cov_names = self.original_cov_names
 
+    def add_line(self, label, values, location=LineLocation.BODY_BOTTOM):
+        """
+        Add a custom line to the table.
+
+        At each location, lines are added in the order at which this method is called.
+        To remove lines, modify the custom_lines[location] attribute.
+
+        Parameters
+        ----------
+        label : str
+            Name of the new line (left-most column).
+        values : list of str
+            List containing the custom information (one entry per model).
```

Comment:
Maybe ``information`` -> ``content``, ``entry`` -> ``item``

Reply:
Agreed, that is clearer!